project/Merging.scala

```diff
@@ -4,6 +4,13 @@ object Merging {
   def customMergeStrategy(oldStrategy: (String) => MergeStrategy): (String => MergeStrategy) = {
     case x if x.endsWith("Resource$AuthenticationType.class") => MergeStrategy.first
     case x if x.endsWith("module-info.class")                 => MergeStrategy.discard
+    // For bouncycastle merge error:
```

Comment:
To be honest, this merge stuff confuses me. Apparently bouncycastle produces multiple jars that all have the some of the same files? The source one below is needed for this sort of thing:

```
[error] Deduplicate found different file contents in the following:
[error]   Jar name = bcprov-ext-jdk18on-1.78.jar, jar org = org.bouncycastle, entry target = org/bouncycastle/crypto/agreement/jpake/JPAKEParticipant.class
[error]   Jar name = bcprov-jdk18on-1.78.jar, jar org = org.bouncycastle, entry target = org/bouncycastle/crypto/agreement/jpake/JPAKEParticipant.class